tests/examples/opinions/united_states/dcd_example.compare.json

```diff
@@ -6,6 +6,7 @@
     "precedential_statuses": "Published",
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
+    "docket_document_numbers": "456",
```

Comment:
I see we have this field now but im not where it's used in our db model.

Reply:
I think that problem is bigger than this single ticket, I am just trying to standardize what already exists.